sungrow/integrationpluginsungrow.json

```diff
@@ -0,0 +1,272 @@
+{
+    "name": "Sungrow",
+    "displayName": "Sungrow",
+    "id": "250c9b83-1127-4013-bbd0-11e7ea482057",
+    "vendors": [
+        {
+            "name": "sungrow",
+            "displayName": "Sungrow",
+            "id": "cdc58e0d-bfdb-45d9-b961-9c0b036c35aa",
+            "thingClasses": [
+                {
+                    "name": "sungrowInverterTcp",
+                    "displayName": "Sungrow Inverter",
+                    "id": "59cb2da4-da07-11ee-adea-7397f8a9afe9",
+                    "createMethods": ["discovery"],
+                    "discoveryType": "weak",
```

Comment:
Is it weak? We cannot be sure we have discovered a Sungrow inverter in the discovery?